<launch>
<rosparam file="$(find robot_control)/cfg/sas_operator_side_receiver.yaml" command="load" ns="sas_operator_side_receiver"/>
<rosparam file="$(find robot_control)/cfg/sas_patient_side_manager.yaml" command="load" ns="sas_patient_side_manager"/>
<!-- <rosparam file="$(find robot_control)/cfg/vitreoretinal_kinematics.yaml" command="load" ns="vitreoretinal_kinematics"/> -->

<node pkg="sas_operator_side_receiver" type="sas_operator_side_receiver_udp_node" name="sas_operator_side_receiver"/>
<node pkg="sas_patient_side_manager" type="sas_patient_side_manager_node" name="sas_patient_side_manager"/>

<node pkg="robot_control" type="store_node.py" name="store"/>
<node pkg="sas_datalogger" type="sas_datalogger_node.py" name="sas_datalogger"/>

<node pkg="robot_control" type="tele_operation_icra2023_ver.py" name="tele_controller"/>

</launch>
